packages/core-bookmarks/bookmarks.js

```diff
@@ -1,7 +1,50 @@
 import './db/helpers';
+import DataLoader from 'dataloader';
 import runMigrations from './db/schema';
+import { Bookmarks } from './db/collections';
 
 export * from './db/collections';
-export default () => {
+
+export const services = {};
+
+// eslint-disable-next-line
+export default (config) => {
   runMigrations();
+  // eslint-disable-next-line
+  return (req) => {
```

Comment:
req is unused.

I also do not understand, why it is wrapped in a req closure. I guess the idea is that the module is re-initialized for every request?

Reply:
well i guess that is the idea for dataloader: you cache per request. so nevermind this comment